scissors-sample/src/main/java/com/lyft/android/scissorssample/MainActivity.java

```diff
@@ -103,6 +112,28 @@ public void onPickClicked() {
                 .pickUsing(this, RequestCodes.PICK_IMAGE_FROM_GALLERY);
     }
 
+    @OnClick(R.id.ratio_fab)
+    public void onRatioClicked() {
```

Comment:
@rharter I was thinking of a `TextView` somewhere I do like the fab icon, to keep it simple a `Toast` should do